analyzer/library/npm/npm.go

```diff
@@ -19,12 +20,18 @@ func init() {
 
 const version = 1
 
-var requiredFiles = []string{"package-lock.json"}
+var requiredFiles = []string{"package-lock.json", "package.json"}
```

Comment:
IMO: If you want to use meaningful strings in multiple places, it is preferable to define them const.